src/python/sdk/examples/trade.py

```diff
@@ -646,8 +747,8 @@ def find_all_fill_events_with_last_taker_order_id(
     index = len(events) - 1
     returns = []
     while index > 0:
-        last_fill_order_id = events[-1]["data"]["taker_order_id"]
-        ev = events[index]
+        last_fill_order_id = events[-1]["data"]["taker_order_id"]  # type: ignore
```

Comment:
I find this comment confusing

If we are supposed to ignore it, can this line be delete? What is going on here?